cryptofeed/gemini/gemini.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, pairs=None, channels=None, callbacks=None, **kwargs):
 
     async def _trade(self, msg, timestamp):
         price = Decimal(msg['price'])
-        side = BID if msg['makerSide'] == 'bid' else ASK
+        side = BUY if msg['makerSide'] == 'bid' else SELL
```

Comment:
@bmoscon Can you chack this works correctly? I don't have access to Gemini where I am from but it looks to me as if this should be `side = BUY if msg['makerSide'] == 'ask' else SELL`,

Reply:
Looks right and matches the docs. I'll give it a test today